mesonbuild/modules/gnome.py

```diff
@@ -22,22 +22,34 @@
 from .. import dependencies
 from .. import mlog
 from .. import mesonlib
+from .. import interpreter
 
+glib_version = None
 girwarning_printed = False
 gresource_warning_printed = False
 
 class GnomeModule:
 
+    def get_glib_version(self, state):
+        global glib_version
+        if glib_version is None:
+            glib_dep = dependencies.PkgConfigDependency('glib-2.0', state.environment, {})
```

Comment:
We do want to use the 'build' version of glib here, because we're going to be running glib-compile-resources on the 'build' machine to generate C code that we then compile. Looking at the dependencies.PkgConfigDependency code, it seems that I can pass 'native=True' to fix this?